handlers/config.go

```diff
@@ -56,7 +55,6 @@ type PublicConfig struct {
 	IstioNamespace           string                          `json:"istioNamespace,omitempty"`
 	IstioComponentNamespaces config.IstioComponentNamespaces `json:"istioComponentNamespaces,omitempty"`
 	IstioLabels              config.IstioLabels              `json:"istioLabels,omitempty"`
-	IstioTelemetryV2         bool                            `json:"istioTelemetryV2"`
```

Comment:
So I guess there must be a front-end PR ?

Reply:
Mmm note that it's also modified in the other PRs used for Mixer removal, I guess here there will be a minor conflict between both efforts.

Reply:
Right, I will rebase and I think we'll be OK.